config/prometheus/rbac-authorization-role.yaml

```diff
@@ -0,0 +1,10 @@
+# Reference link: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  # 'namespace' is omitted as 'ClusterRole'-s are not namespaced
+  name: lvm-operator-controller-metric-rbac-role
+rules:
+- apiGroups: [""] # "" indicates the core API group
+  resources: ["services", "endpoints", "pods", "nodes"]
```

Comment:
do we need  `nodes`?  Don't see any metrics being scraped from the `nodes` resources. Or maybe I'm missing something.

Reply:
idea was, if we require anything from nodes, then we don't have to change any rbac rules.
Removed `nodes`